roles/foreman_installer/tasks/upgrade.yml

```diff
@@ -1,16 +1,19 @@
 ---
 - name: 'Stop services'
   command: foreman-maintain service stop
+  when: foreman_installer_scenario != 'foreman'
 
 - name: 'Update packages'
   import_role:
     name: update_os_packages
 
 - set_fact:
     foreman_installer_options_internal_use_only: "{{ [ '--upgrade', '--certs-update-all' ] + foreman_installer_options_internal_use_only }}"
-  when: foreman_installer_version_file['content'] | b64decode is version('2.1', '<')
+  when:
+    - foreman_installer_version_file['content'] | b64decode is version('2.1', '<')
+    - foreman_installer_scenario != 'foreman'
 
 - name: 'Run installer upgrade'
   import_tasks: "install.yml"
   vars:
-    foreman_installer_disable_system_checks: true
+    foreman_installer_disable_system_checks: foreman_installer_scenario != 'foreman'
```

Comment:
Does this work outside of curly brackets?

Reply:
I'll admit I only tested this with a Foreman pipeline, but there it evaluated to `False` and got it to upgrade me to 2.1.0-develop (other than that the upgrade itself failed due to HTTP 503).